backend/src/services/linear/api/resolvers.py

```
@@ -15230,3 +15453,487 @@ def resolve_teamMembershipUpdate(obj, info, **kwargs):
+        )
+
+        session.add(workflow_state)
+
+        # Return WorkflowStatePayload structure
+        return {
+            "lastSyncId": 0.0,  # In a real implementation, this would come from a sync tracking system
+            "success": True,
+            "workflowState": workflow_state,
+        }
+
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)